### ISSUE TITLE
win32 build not installing correctly (Trac #1062)

### ISSUE THREAD

**Issue by butlerpd** [SasView/sasview#1118]
Rob Dalgliesh reports that the Jan 19, 2018 build of !SasView won't install. Instead the error message attached is presented.  I reproduced this error downloading the win7 build from the master builds tab.  This however is currently the 32 bit build.  It attempts to install on the x86 program folder and then throws this error.  If however I download the 64 bi win7 version (from the "ALL" builds tab ... and not the PR win7 build).  the system installs with no issues in program files.

Depending on the cause this may be moot if we do not release a 32 bit version but until we decide that I will label this blocker. 

Migrated from http://trac.sasview.org/ticket/1062
```json
{
    "status": "closed",
    "changetime": "2018-02-04T23:15:55",
    "_ts": "2018-02-04 23:15:55.281092+00:00",
    "description": "Rob Dalgliesh reports that the Jan 19, 2018 build of !SasView won't install. Instead the error message attached is presented.  I reproduced this error downloading the win7 build from the master builds tab.  This however is currently the 32 bit build.  It attempts to install on the x86 program folder and then throws this error.  If however I download the 64 bi win7 version (from the \"ALL\" builds tab ... and not the PR win7 build).  the system installs with no issues in program files.\n\nDepending on the cause this may be moot if we do not release a 32 bit version but until we decide that I will label this blocker. ",
    "reporter": "butler",
    "cc": "",
    "resolution": "worksforme",
    "workpackage": "SasView Bug Fixing",
    "time": "2018-01-22T22:17:46",
    "component": "SasView",
    "summary": "win32 build not installing correctly",
    "priority": "blocker",
    "keywords": "",
    "milestone": "SasView 4.2.0",
    "owner": "piotr",
    "type": "defect"
}
```

**Comment by rozyczko** [comment]
no longer relevant. SV 5.0 is 64 bit only

**Comment by butlerpd** [comment]
Trac update at `2018/01/22 22:18:42`: 
* **butler** changed attachment from "" to "InstallError.png"
* **butler** commented:

installation error message

**Comment by rozyczko** [comment]
Trac update at `2018/01/23 13:30:00`: 
* **piotr** changed owner from "" to "piotr"
* **piotr** changed status from "new" to "accepted"

**Comment by rozyczko** [comment]
Trac update at `2018/01/23 13:35:16`: **piotr** commented:

Let me understand it:

32-bit Sasview installer on Windows 64: works locally as expected on a number of hosts.
64-bit Sasview installer on Windows 64: works locally as expected on a number of hosts.

The attached image is from a windows 64 machine (see "c:\program files (x86)" which exists only on win64).
This means the 32 bit installer should work.

If you (Paul) managed to reproduce the problem, please let me know the installer build date, your OS specs and confirm the failure point.

**Comment by RichardHeenan** [comment]
Trac update at `2018/01/23 16:23:44`: **richardh** commented:

Tried the 32 bit Win7 installer from Jenkins on my office PC, which is windows 7, all worked fine for me.

Tried install to 2 different directories, including one in Program Files x86.

Richard (who shares an office with Rob D who first reported this)

**Comment by gonzalezma** [comment]
Trac update at `2018/01/24 09:17:17`: **gonzalezm** commented:

Replying to [comment:3 richardh]:
> Tried the 32 bit Win7 installer from Jenkins on my office PC, which is windows 7, all worked fine for me.
> 
> Tried install to 2 different directories, including one in Program Files x86.
> 
> Richard (who shares an office with Rob D who first reported this)


The 32-bit install worked also fine for me, on a Windows 8.1 PC.

**Comment by RichardHeenan** [comment]
Trac update at `2018/01/29 10:48:21`: **richardh** commented:

Following comments over weekend from Paul B who still sees failures, I checked again today on my desktop Win7 64 bit machine.
This time I uninstalled all mentions of (very old???) sasview in add/remove programs, and deleted program files (x86) \sasview directory.
From Jenkins the latest win7 32bit installer number 36 of 18 days ago, works fine for me, in the default install directory.

Richard

**Comment by butlerpd** [comment]
Trac update at `2018/02/04 23:15:55`: 
* **butler** commented:

While reported on Sunday Jan 28 the last test I did and reported failing was Jan 23 so using build from before that.  Today, Sunday Feb 4 I have used the Feb 2 32 build on jenkins which now seems to install on all 3 machines I've tried (a windows 7 laptop, an old AMD 64bit running windows 10 Home edition and a virtual machine running windows server 2016 enterprise) ... and all three install just fine.  I have not tested my office windows 7 machine again but since these did not work before and everyone else now gets it to work am going to assume that whatever the glitch it has been magically fixed (assuming Rob D also has it working now since test by office mate Richard Heenan says all his tests now show it is working).

So going to close now 


* **butler** changed resolution from "" to "worksforme"
* **butler** changed status from "accepted" to "closed"
